development-tools/clinic.rst

```diff
@@ -1996,6 +1996,36 @@ The generated glue code looks like this:
       return return_value;
    }
 
+You can lock up to two additional objects
+by supplying their C variable names as arguments
+to the ``@critical_section`` directive.
+This example from :cpy-file:`Modules/_weakref.c` takes
+one additional argument (a C variable named ``object``)::
+
+   /*[clinic input]
+   @critical_section object
+   _weakref.getweakrefcount -> Py_ssize_t
+
+   object: object
```

Comment:
Maybe it would be better to find an example that uses a different name, in order to avoid confusion between the variable name and its type.

Reply:
Well, it does not have to be a real example :) The Argument Clinic docs are full of code examples that are just made up.